### PR TITLE
Refresh lockfiles

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Refresh lockfiles using `dotnet restore -f --force-evaluate --no-cache`. Only change should be the internal bitwarden projects which will use the latest version instead of whichever was previously defined.

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: bitwarden_license/src/Commercial.Core/packages.lock.json [hunk 1]
```diff
@@ -2424,43 +2424,43 @@
...
-          "SendGrid": "9.27.0",
-          "Sentry.Serilog": "3.16.0",
-          "Serilog.AspNetCore": "5.0.0",
-          "Serilog.Extensions.Logging": "3.1.0",
-          "Serilog.Extensions.Logging.File": "2.0.0",
-          "Serilog.Sinks.AzureCosmosDB": "2.0.0",
-          "Serilog.Sinks.SyslogMessages": "2.0.6",
-          "Stripe.net": "40.0.0",
-          "YubicoDotNetClient": "1.2.0"
+          "AWSSDK.SQS": "[3.7.2.47, )",
```
Comment (JaredSnider-Bitwarden) [inline bitwarden_license/src/Commercial.Core/packages.lock.json:2427]: ❓ is it safe to go from exact version specification to use exclusive bound specification (allowing the use of any newer versions than the currently specified one)?

Comment (Hinton) [reply bitwarden_license/src/Commercial.Core/packages.lock.json:2427]: While they look different behavior wise are identical. https://learn.microsoft.com/en-us/nuget/concepts/package-versioning.

To lock to exact versions you need `[version]`.

Comment (JaredSnider-Bitwarden) [reply bitwarden_license/src/Commercial.Core/packages.lock.json:2427]: Ah, I'm sorry. Mixed that up. Thank you for the link and clarification.

**Review by withinfocus — APPROVED** [review]
Did a quick scan but this is generated content.

From a DevOps and build perspective, the inclusion of inter-project versioning in these e.g. `Core` is maybe something we need to automate with version bumps. I will mention it to them.

**Review by JaredSnider-Bitwarden — APPROVED** [review]
👍

**Pull request merged**